src/frontend/pages/general/SignIn/assets/waterloop-cms-logo.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="321" height="42" viewBox="0 0 321 42" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
TODO: Design for this logo needs to be changed to use "Executive Dashboard" instead of "CMS"